utils/base16384/test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+[[ "1234567890abcdefg" == `echo 1234567890abcdefg | base16384 -e - - | base16384 -d - -` ]]
```

Comment:
```suggestion
#!/bin/sh

[ "1234567890abcdefg" = "$(echo 1234567890abcdefg | base16384 -e - - | base16384 -d - -)" ]
```

openwrt uses busybox sh and bash is not included by default.